extensions/kubernetes-service-binding/runtime/src/test/java/io/quarkus/kubernetes/service/binding/runtime/ServiceBindingTest.java

```diff
@@ -40,4 +43,19 @@ void testK8s() {
         assertThat(binding.getProperties()).containsExactly(entry("test-secret-key", "test-secret-value"));
     }
 
-}
+    @Test
+
```

Comment:
Unnecessary new line.

Reply:
Reverted this change as well as the class under test is reverted.